modules/registry-configuring-storage-azure-user-infra.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+//* registry/configuring-registry-storage-azure.adoc
+
+[id="registry-configuring-storage-azure-user-infra_{context}"]
+= Configuring registry storage for Azure
+
+During installation, your cloud credentials are sufficient to create Azure Blob
+Storage and the Registry Operator will automatically configure storage. Azure
+registry storage uses the struct, `ImageRegistryConfigStorageAzure`.
```

Comment:
Is there any place where customers can see `ImageRegistryConfigStorageAzure`?

Reply:
s/struct, `ImageRegistryConfigStorageAzure`./`ImageRegistryConfigStorageAzure` struct.